### PR TITLE
SP-19172 - add format field to RawImageContainer

### PR DESCRIPTION
**Pull request by inspector-code**

### CODE REVIEW

File: p-ping.yml [hunk 1]
```diff
@@ -16,11 +16,11 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/DeviceInfo"
+                $ref: "#/components/schemas/DeviceInfoDeprecated"
```
Comment (ikliashchou) [inline p-ping.yml:19]: why?

File: rt-raw-image.yml [hunk 2]
```diff
@@ -4,14 +4,26 @@ components:
 
     DocumentImageResult:
```
Comment (ikliashchou) [inline rt-raw-image.yml:5]: this is all wrong
this should be the same type for both RPRM_ResultType_RawUncroppedImage =16 and RPRM_ResultType_RawImage = 1 results.

**Review by ikliashchou — CHANGES_REQUESTED** [review]

File: rt-raw-image.yml [hunk 3]
```diff
@@ -4,11 +4,11 @@ components:
...
       title: "Document Image"
+      description: "Contains cropped and rotated with perspective compensation image of document.
+        Single input image can contain multiple document side/pages, which will be returned as separated results.
+        Most of coordinates in other types defined on that image"
```
Comment (vyakimchik) [inline rt-raw-image.yml:9]:
```suggestion
      description: "Contains document images after processing."
```

File: rt-raw-image.yml [hunk 4]
```diff
@@ -4,11 +4,9 @@ components:
 
     DocumentImageResult:
       title: "Document Image"
+      description: "Contains document images after processing."
```
Comment (ikliashchou) [inline rt-raw-image.yml:7]:
```suggestion
      description: "Contains document image."
```

**Review by ikliashchou — APPROVED** [review]

**Pull request merged**